### DISCUSSION TITLE
Filter away boring content (aka filter by title, description and media)

### DISCUSSION THREAD

**Discussion by henrik242** [SauceLLC/sauce4strava#203] [General]
I'm looking for a way to filter boring content, i.e. activities that use the default title (e.g. "Morning Ride"), has no description, no uploaded media and less than an hour.

Would that be difficult to achieve?

**Comment by mayfield** [comment]
Some of these might be possible.  The info available in this context is a bit limited but this gives me some ideas.

**Reply by henrik242** [reply]
Cool!

**Comment by henrik242** [comment]
@mayfield How's the progress going?

**Comment by mayfield** [comment]
Added some new* social filters.

**Reply by henrik242** [reply]
Thanks! Much appreciated 👍👍

**Reply by henrik242** [reply]
@mayfield Are these documented somewhere? Couldn't immediately find something in the UI

EDIT: I guess the version in the app store (8.5.7) might be too old for the latest features 🤔

**Comment by mayfield** [comment]
8.5.7 should have them.  Add a new filter and look for the new "Social" group.  And then under category you can now select "None".  So you can create a filter of "Has Kudos" -> "None" -> "Hide" to hide activities with no kudos for example.

**Reply by henrik242** [reply]
Thanks! I'm afraid that doesn't cover the issue I have (i.e. hiding boring content), but still nice to have.

Examples:
 - An activity with a non-default title isn't boring
 - An activity with pictures isn't boring
 - An activity with a description isn't boring
